production/scrapers/oklahoma_manual.R

```diff
@@ -0,0 +1,90 @@
+            `Inmate Total Currently Positive`,
+            `Employee Total Tested`, 
+            `Employee Total Positive`, 
+            `Employee Total Currently Positive`
+        )
+}
+
+oklahoma_manual_extract <- function(x){
+    x %>%
+        rename("Residents.Deaths" = "Inmate Deaths by Facility", 
+               "Residents.Tadmin" = "Inmate Total Tested", 
+               "Residents.Confirmed" = "Inmate Total Positive", 
+               "Residents.Active" = "Inmate Total Currently Positive", 
+               "Staff.Tadmin.Drop_" = "Employee Total Tested", 
+               "Staff.Confirmed" = "Employee Total Positive", 
+               "Staff.Active.Drop_" = "Employee Total Currently Positive") %>% 
+        select(!ends_with(".Drop_")) %>% 
+        clean_scraped_df() %>% 
+        # To avoid double-counting, statewide deaths should be NA 
+        # These are reported at the facility-level instead 
+        # This should be reflected in the manual entry, so this is just to be safe  
+        mutate(Residents.Deaths = ifelse(Name == "STATEWIDE", NA, Residents.Deaths))
+}
+
+#' Scraper class for Oklahoma COVID data
+#' 
+#' @name oklahoma_manual_scraper
+#' @description Oklahoma's dashboard isn't machine-readable, so we manually extract 
+#' the relevant information. 
+#' \describe{
+#'   \item{Name}{The facility name.}
+#'   \item{Inmate Deaths by Facility}{Facility-level cumulative deaths}
+#'   \item{Inmate Total Tested}{Statewide-level total tests administered}
+#'   \item{Inmate Total Positive}{Statewide-level cumulative COVID cases for inmates}
+#'   \item{Inmate Total Currently Positive}{Statewide-level current active COVID infections}
+#'   \item{Employee Total Positive}{Statewide-level cumulative COVID cases for staff}
+
+oklahoma_manual_scraper <- R6Class(
+    "oklahoma_manual_scraper",
+    inherit = generic_scraper,
+    public = list(
+        log = NULL,
+        initialize = function(
+            log,
+            url = "https://oklahoma.gov/doc/covid-19-stats.html",
+            id = "oklahoma_manual",
+            type = "manual",
+            state = "OK",
+            jurisdiction = "state",
+            # pull the JSON data directly from the API
+            pull_func = oklahoma_manual_pull,
+            # restructuring the data means pulling out the data portion of the json
+            restruct_func = oklahoma_manual_restruct,
+            # Rename the columns to appropriate database names
+            extract_func = oklahoma_manual_extract){
+            super$initialize(
+                url = url, id = id, pull_func = pull_func, type = type,
+                restruct_func = restruct_func, extract_func = extract_func,
+                log = log, state = state, jurisdiction = jurisdiction)
+        }
+    )
+)
+
+if(sys.nframe() == 0){
+    oklahoma_manual <- oklahoma_manual_scraper$new(log=TRUE)
+    oklahoma_manual_scraper$raw_data
+    oklahoma_manual_scraper$pull_raw()
```

Comment:
` oklahoma_manual_scraper$pull_raw()` needs to be ` oklahoma_manual$pull_raw()` as it was assigned on line 79. that should fix the attempt to apply non function

Reply:
Wow I'm dumb, thanks!